Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN for filename in $( \
     done
 
 
-FROM busybox:1.30.1
+FROM alpine:3.7
```

Comment:
I think this should be refactored to just base off the alpine from the beginning. Right now the builder is alpine and then `lib*.so` libs are copied over to busybox. If we want alpine as base image this is unnecessary.

Reply:
Trying to keep the changes to a minimum, so we don't introduce more changes than required

Reply:
Trying to keep changes to a minimum so we don't introduce any issues

Reply:
Approved